copier.yml

```diff
@@ -12,7 +12,8 @@ _envops:
 # Other Copier configurations
 _exclude:
   - /.git
-  - /.github
+  - /.github/workflows/lint.yml
```

Comment:
I think that you can simply remove this line and the `pre-commit.yml.jinja` file above. This way, the same `lint.yml` will be used here in the template and in the subprojects. After all, pre-commit should work exactly equally for all.

Reply:
I'd rather keep the files separate. Otherwise one day, someone will shot him.herself in the foot.

Reply:
We can change it that day :blush:

Reply:
No seriously. People will not notice this file is shared and break one or the other.

Reply:
:man_shrugging: ok